src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/PiglinListener.java

```diff
@@ -38,6 +48,30 @@ public void onEntityPickup(EntityPickupItemEvent e) {
         }
     }
 
+    @EventHandler
+    public void onPiglinDropItem(EntityDropItemEvent e) {
+        if (e.getEntity() instanceof Piglin) {
+            Piglin piglin = (Piglin) e.getEntity();
```

Comment:
You make an Piglin variable but then only use it once (and use e.getEntity another). You also don't actually even use any methods in this class...

Reply:
made piglin a variable based on a request from TheBusyBiscuit...  So should I or should I not make a variable? or just use e.getEntity()

Reply:
Having a variable is fine but, you just use the Piglin class for no reason. You also only use it once and then still have an e.getEntity

Use the variable everywhere or use the longer calls.